data/countries/GU.yaml

```diff
@@ -1,5 +1,7 @@
 holidays:
   # @attrib https://en.wikipedia.org/wiki/Public_holidays_in_the_United_States#Guam
+  # @attrib https://en.wikipedia.org/wiki/Public_holidays_in_Guam
+  # @source https://www.guamtax.com/help/holidays.html
```

Comment:
Two other sources for you:

* http://www.guamcourts.org/CompilerofLaws/GCA/01gca/1gc010.PDF - by far the best and most official list of Guam government holidays that I have found.
* http://www.guamcourts.org/Holidays-and-Hours/JOG-Holidays-and-Hours.html - when the Judiciary of Guam has holidays.

Reply:
I found those two by reading about the government of Guam and how it relates to the US which lead me to https://en.wikipedia.org/wiki/Legislature_of_Guam .